plugin.go

```diff
@@ -51,10 +52,10 @@ func (p Plugin) Exec() error {
 
 	cmd := exec.Command("sonar-scanner", args...)
 	// fmt.Printf("==> Executing: %s\n", strings.Join(cmd.Args, " "))
-	output, err := cmd.CombinedOutput()
-	if len(output) > 0 {
-		fmt.Printf("==> Code Analysis Result: %s\n", string(output))
-	}
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
I'm not sure, how the stderr of plugins is handled, it might worth a trial. Since the stderr of sonar-scanner was written to the stdout of the plugin, my idea was to keep this behavior. I'll give a try to os.Stderr.